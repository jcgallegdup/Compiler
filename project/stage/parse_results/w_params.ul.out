Insert key=Implement me! val=AST.Function@76ed5528
Insert key=Implement me! val=AST.Function@2c7b84de
Insert key=Implement me! val=AST.Function@3fee733d
Insert key=Implement me! val=AST.Function@5acf9800
Insert key=Implement me! val=AST.Function@4617c264
Insert key=Implement me! val=AST.Function@36baf30c
Insert key=Implement me! val=AST.Function@7a81197d
Error:0(line):0(position): Could not find function with name 'main' and type 'void'.
