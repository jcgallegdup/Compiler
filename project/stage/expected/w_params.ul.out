void fO_123456789_Xa(int asdf_kdf23_, int[89] MEOW_meow_1094385, int[1010] _meow_1094385, int[8912] _1094385)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
char f12345098_234fXx_(char lowercase, float UPPERCASE, boolean ______, void _1234567890)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
float _O_123456789(boolean AAas_2394dkKJD, float __Ka235, string _KDJ___123, float __Ka235, string _KD1J___123, float __Ka235, string _KDJ___123)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
int[100] _123456_FooZ(float _____1, void __KDkk1236, string __3426Kka)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
int _____123567FkoP(string __123Kk, string __234IKk, string __234IKk, string __234IKk, int UPPERCASE)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
string f12345098_234fXx_(void askj_234wf, int D0_1234Kkk)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
void f12345098_234fXx_(int[234] _234JKDd, int ___1Kk, string _123KkksldjbB_)
{
    void fO_123456789_Xa;
    int[8912] _1094385;
    char f12345098_234fXx_;
    float _____1;
    void fO_123456789_Xa;
}
